commands/drive/robotrelativedrive.py

```diff
@@ -29,6 +29,6 @@ def execute(self) -> None:
             self.forward(),
             self.sideways(),
             self.rotation()
-            * Preferences.getFloat("Robot Relative Sensitivity", 0.2),  # better control
+            * Preferences.getFloat("Robot Relative Sensitivity", 0.8),  # better control
```

Comment:
Out of curiosity, is this just something you adjusted by feel on the actual bot?

Reply:
yeah, we don't use robot relative very often (if ever) so having higher defaults makes more sense for a systems test, the preferences allow it to be tuned down